index.html

```diff
@@ -2833,136 +2833,105 @@ <h3>Consumption</h3>
-documents</a> that are expressed as JSON-LD. JSON-LD <a>DID documents</a>
-are indicated by a <code>contentType</code> resolver metadata field that is
-set to <code>application/did+ld+json</code>.
-      </p>
-
-      <p class="issue atrisk" title="IETF did+ld+json media type registration">
-Use of the media type <code>application/did+ld+json</code> is pending
-clarification over the registration of
-<a href="https://datatracker.ietf.org/doc/html/draft-w3cdidwg-media-types-with-multiple-suffixes">
-media types with multiple suffixes</a>. The alternative will be to use
-<code>application/ld+json</code> with an expected profile parameter of
-<code>https://www.w3.org/ns/did/json-ld-profile</code> if multiple suffixes
-cannot be registered by the time the rest of DID Core is ready for W3C
-Proposed Recommendation. See also <a href="https://github.com/w3c/did-core/issues/208">
-Issue 208</a>.
+JSON-LD [[JSON-LD11]] is a JSON-based format used to serialize <a
+href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. This
+section defines the <a>production</a> and <a>consumption</a> rules for the
+JSON-LD <a>representation</a>.
       </p>
 
       <p>
-The following application-specific modifications are made by this specification
-to the JSON-LD specification [[JSON-LD11]] to ease interoperability between
-JSON and JSON-LD implementations:
+The JSON-LD <a>representation</a> defines the following representation-specific
+entries:
       </p>
 
-      <ul>
-        <li>
-The <code>@id</code> and <code>@type</code> keywords are aliased to
-<code><a>id</a></code> and <code>type</code> respectively, enabling developers
-to use this specification as idiomatic JSON.
-        </li>
-        <li>
-Even though JSON-LD allows any IRI as node identifiers, <a>DID documents</a>
-are explicitly restricted to only describe <a>DIDs</a>. This means that the value of
-<code><a>id</a></code> that refers to the <a>DID subject</a> MUST be a valid
-<a>DID</a> and not any other kind of IRI.
-        </li>
-        <li>
-Data types, such as integers, dates, units of measure, and URLs, are
-automatically typed to provide stronger type guarantees for use cases that
-require them.
-        </li>
-      </ul>
+      <dl>
+        <dt><dfn>@context</dfn></dt>
+        <dd>
+The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a>
+is either a <a data-cite="INFRA#string">string</a> or a <a
+data-cite="INFRA#list">list</a> containing any combination of <a
+data-cite="INFRA#string">strings</a> and/or <a data-cite="INFRA#maps">ordered
+maps</a>.
+        </dd>
+      </dl>
 
       <section>
         <h3>Production</h3>
 
         <p>
-The <a>DID document</a> MUST be serialized according to the
-<a href="#production">production rules for JSON</a>, with one additional
-requirement: The serialized <a>DID document</a> MUST include the
-<code>@context</code> member.
+All data structures expressed by the <a href="#data-model">data model</a> MUST
```

Comment:
As I noted in https://github.com/w3c/did-core/pull/644#discussion_r576329987, it would be a major change to now apply the representations to other things than DID documents. The DID resolution functions and data structures were intentionally left abstract, for good reasons; let's not change that now.

Reply:
I don't read this as changing the requirements... resolution returns an INFRA map with at least an `id`... you can't say anything other than that.... and have it still return a "DID Document" :)

Reply:
> The DID resolution functions and data structures were intentionally left abstract, for good reasons; let's not change that now.

The new DID Resolution section made them concrete by saying the following: 

> If the resolution is successful, and if the resolve function was called, this MUST be a DID document abstract data model (a map) as described in § 4. Data Model that is capable of being transformed into a conforming DID Document (representation), using the production rules specified by the representation.

and

> The entire metadata structure MUST be serializable in a conforming representation.

I was very careful to make sure the language didn't preclude other ways of working with the didDocument or metadata (when dealing with it in programming environments) or requiring it to be serialized in a specific representation (serializing it to HTTP is fine, as long as you can ALSO serialize the same data to a representation).

This is not a big change... it allows you to keep things abstract if you want to (as long as there is a mechanism to get it to a representation -- and that makes sure everything in the abstract in DID Core is also serializable).

Reply:
> The entire metadata structure MUST be serializable in a conforming representation.

I will note that there was an objection to adding this language (see https://github.com/w3c/did-core/pull/601#discussion_r573031499), and that this language was added without addressing the objection, and with very little review by the WG.

Reply:
That objection was resolved here: https://github.com/w3c/did-core/pull/601#issuecomment-777092645